Examples do not work

I'm trying to run the examples but keep getting the error below:
```
suricactus@orange:~/work/tests/redux-saga-requests/examples/basic$ node --version
v10.3.0
suricactus@orange:~/work/tests/redux-saga-requests/examples$ yarn
yarn install v1.7.0
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.2.4: The platform "linux" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...

success Saved lockfile.
Done in 3.98s.
suricactus@orange:~/work/tests/redux-saga-requests/examples$ cd basic/
suricactus@orange:~/work/tests/redux-saga-requests/examples/basic$ npm start

> redux-saga-requests-basic-example@1.0.0 start /home/suricactus/work/tests/redux-saga-requests/examples/basic
> webpack-dev-server

ℹ ｢wds｣: Project is running at http://localhost:3000/
ℹ ｢wds｣: webpack output is served from /
✖ ｢wdm｣: Hash: 0aa3479e85fb680c205f
Version: webpack 4.9.1
Time: 1765ms
Built at: 07/01/2018 2:41:31 AM
 1 asset
Entrypoint main = main.js
[../node_modules/babel-polyfill/lib/index.js] 833 bytes {main} [built]
[../node_modules/loglevel/lib/loglevel.js] 7.68 KiB {main} [built]
[../node_modules/webpack/hot sync ^\.\/log$] ../node_modules/webpack/hot sync nonrecursive ^\.\/log$ 170 bytes {main} [built]
[0] multi ../node_modules/webpack-dev-server/client?http://localhost:3000 webpack/hot/dev-server ./src 52 bytes {main} [built]
[../node_modules/strip-ansi/index.js] 161 bytes {main} [built]
[../node_modules/url/url.js] 22.8 KiB {main} [built]
[../node_modules/webpack-dev-server/client/index.js?http://localhost:3000] ../node_modules/webpack-dev-server/client?http://localhost:3000 7.75 KiB {main} [built]
[../node_modules/webpack-dev-server/client/overlay.js] 3.58 KiB {main} [built]
[../node_modules/webpack/hot/dev-server.js] 1.66 KiB {main} [built]
[../node_modules/webpack/hot/emitter.js] 77 bytes {main} [built]
[../node_modules/webpack/hot/log-apply-result.js] 1.31 KiB {main} [built]
[../node_modules/webpack/hot/log.js] 1.03 KiB {main} [built]
[./src/components/app.js] 3.57 KiB {main} [built]
[./src/index.js] 507 bytes {main} [built]
[./src/store/index.js] 1.89 KiB {main} [built]
    + 444 hidden modules

ERROR in ./src/store/index.js
Module not found: Error: Can't resolve 'redux-saga-requests' in '/home/suricactus/work/tests/redux-saga-requests/examples/basic/src/store'
 @ ./src/store/index.js 8:0-75 22:17-38 26:17-30
 @ ./src/index.js
 @ multi ../node_modules/webpack-dev-server/client?http://localhost:3000 webpack/hot/dev-server ./src

ERROR in ./src/store/reducers.js
Module not found: Error: Can't resolve 'redux-saga-requests' in '/home/suricactus/work/tests/redux-saga-requests/examples/basic/src/store'
 @ ./src/store/reducers.js 3:0-54 7:26-41 12:25-40
 @ ./src/store/index.js
 @ ./src/index.js
 @ multi ../node_modules/webpack-dev-server/client?http://localhost:3000 webpack/hot/dev-server ./src

ERROR in ./src/store/index.js
Module not found: Error: Can't resolve 'redux-saga-requests-axios' in '/home/suricactus/work/tests/redux-saga-requests/examples/basic/src/store'
 @ ./src/store/index.js 9:0-52 22:64-75
 @ ./src/index.js
 @ multi ../node_modules/webpack-dev-server/client?http://localhost:3000 webpack/hot/dev-server ./src
Child html-webpack-plugin for "index.html":
     1 asset
    Entrypoint undefined = index.html
    [../node_modules/html-webpack-plugin/lib/loader.js!./src/index.html] 500 bytes {0} [built]
    [../node_modules/lodash/lodash.js] 527 KiB {0} [built]
    [../node_modules/webpack/buildin/global.js] 509 bytes {0} [built]
    [../node_modules/webpack/buildin/module.js] 519 bytes {0} [built]
ℹ ｢wdm｣: Failed to compile.
```
I've also tried installing with npm in each example directory or switching to node 8, but I kept getting this error.

--- a/examples/README.md
+++ b/examples/README.md
@@ -1,5 +1,13 @@
 # Examples
 
+For examples to work make sure you ran the following commands in the project root
+
+```
+yarn run install
+yarn run build
+```
+
+
 Examples in this package take advantage of [Yarn Workspaces](https://yarnpkg.com/en/docs/workspaces), so you can install
 `node_modules` for all examples with just one `yarn install` command from this directory. If you don't use `yarn`, that's
 fine too, just run `npm install` in an example directory, as usual.

